bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -811,7 +811,7 @@ public void setupFfmpegFormat(FFmpegFormat format) {
             logger.warn("The camera tried to use a FFmpeg feature when the location for FFmpeg is not known.");
             return;
         }
-        if (rtspUri.toLowerCase().contains("rtsp")) {
+        if (mjpegUri.toLowerCase().startsWith("rtsp") || rtspUri.toLowerCase().startsWith("rtsp")) {
```

Comment:
Should you rather consider "rtsp://"? Later in that method, you use "rtsp://".

Reply:
Have added the suggestion.